app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextModuleActions.java

```diff
@@ -204,13 +228,15 @@ else if (((selectionEnd <= (_hlEditor.length() - _action.length())) &&
             {CommonTextModuleActions.ACTION_DELETE_LINES_ICON, 0},
             {CommonTextModuleActions.ACTION_OPEN_LINK_BROWSER__ICON, 1},
             {R.drawable.ic_link_black_24dp, 2}, {R.drawable.ic_image_black_24dp, 3},
-            {CommonTextModuleActions.ACTION_SPECIAL_KEY__ICON, 4},
-            {R.drawable.ic_keyboard_return_black_24dp, 5},
+            {R.drawable.ic_add_a_photo_black_24dp, 4},
```

Comment:
just leave the existing and add the new ones, will order accordingly if needed. (currently looks like theres something missing)